ebuild-writing/variables/text.xml

```diff
@@ -588,5 +588,54 @@ when joining paths. For example: <c>${D%/}/</c>, <c>${ED%/}/</c>,
 </body>
 </section>
 
+<section>
+<title>Use of constant-value variables in ebuilds</title>
+<body>
+
+<p>
+Variables frequently have significant value in ebuilds, making it
+possible to avoid unnecessary repetitions and make maintenance easier.
+However, references to constant-value variables should be used with care
+as their proactive use can harm readability and increase maintenance
+burden (e.g. when renaming the package). In particular, using variables
+whose values are not strictly guaranteed to match the expected string
+should be avoided.
```

Comment:
I've read this sentence a few times and I'm not sure I understand. Could you please clarify?

Reply:
Here's my take on this sentence. I'm not sure if I've made it more complex and simpler: `In particular, using variables whose values have no direct correlation with the expected string other than matching it by coincidence should be avoided.` Maybe following it with a concrete example can clarify it better.

Reply:
I've took the first part of it but stripped the 'other than ...'. I think it's unnecessary given the context, and the rest sums it up well.